### PR TITLE
Improve documentation v4 (LC15)

### LINKED ISSUE #4878: Feature Request: Add App Signing Certificate SHA-256 Hash to README
### Describe the feature

Hello,

I would like to request that the app signing certificate SHA-256 hash be added to the GitHub README for this project. 

### Why would it be useful to add?

This would greatly assist with verifying the authenticity of APKs downloaded from non-trusted sources, such as directly from GitHub or other websites, compared to trusted sources like the Google Play Store or Accrescent.

By including this information, users can easily use tools like [AppVerifier](https://github.com/soupslurpr/AppVerifier) to confirm the APK's authenticity before installation.

### Did the feature exist in Lawnchair v2? (Play Store version)

No

### Example(s)

An example can be seen here:

https://github.com/beemdevelopment/Aegis?tab=readme-ov-file#verification

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by validcube**
## Description

This PR marked the fourth major change to the documentation.

close #4878 

~~Right now this PR needs updating the screenshot to Lawnchair 15 which isn't possible, I'm currently making LC15 demo-able before I have this PR finished.~~

Here's a list of tasks that need to be done to be marked as ready:

### Lawnchair 15 Preview
- [x] #4975 
- [x] #5029
- [x] Match Icon Shape (61c3db874e07a6b99fd2512aaab520a15086f7f8)
- [x] Update the preview

### Developer
- [x] How to update locally bundled Google Fonts listing
- [x] ~~Inform regarding WallpaperManager~~
- [x] Update for https://github.com/LawnchairLauncher/lawnchair/pull/5034

### Miscellaneous
- [x] #4878 

## Type of change

:white_check_mark: General change (non-breaking change that doesn't fit the below categories like copyediting)
:x: Bug fix (non-breaking change which fixes an issue)
:x: New feature (non-breaking change which adds functionality)
:x: Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by validcube** [comment]
Note the screenshot I'm about to update with is based on modified Lawnchair 15 made to be launchable.

Font are still in Roboto. If there are any changes. Please notify.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -51,13 +51,13 @@ run `git submodule update --init --recursive`.
...
 -   Make sure your code is logical and well-formatted. If using Kotlin, see [“Coding conventions” in the Kotlin documentation](https://kotlinlang.org/docs/coding-conventions.html);
--   [The `lawnchair` package](https://github.com/LawnchairLauncher/lawnchair/tree/14-dev/lawnchair) houses Lawnchair’s own code, whereas [the `src` package](https://github.com/LawnchairLauncher/lawnchair/tree/14-dev/src) includes a clone of the Launcher3 codebase with modifications. Generally, place new files in the former, keeping changes to the latter to a minimum.
+-   [The `lawnchair` package](https://github.com/LawnchairLauncher/lawnchair/tree/15-dev/lawnchair) houses Lawnchair’s own code, whereas [the `src` package](https://github.com/LawnchairLauncher/lawnchair/tree/15-dev/src) includes a clone of the Launcher3 codebase with modifications. Generally, place new files in the former, keeping changes to the latter to a minimum.
 
 #### Additional documentation
 
--   The Lawnchair Wiki ([at Github](https://github.com/LawnchairLauncher/lawnchair/wiki))
+-   The Lawnchair Wiki ([GitHub Wiki](https://github.com/LawnchairLauncher/lawnchair/wiki))
```
Comment (Goooler) [inline CONTRIBUTING.md]:
```suggestion
-   The Lawnchair ([GitHub Wiki](https://github.com/LawnchairLauncher/lawnchair/wiki))
```

**Review by Goooler — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (SuperDragonXD) [reply CONTRIBUTING.md]: I think we can leave this text as-is instead.

However, a better approach for consistency is to just directly link the text:

```
* Lawnchair GitHub Wiki
* Lawnchair Visual Guidelines
...
```

**Comment by validcube** [comment]
Update: Unfortunate event, my phone is broken (my pc is too slow for an android emulator) so I can't update the device preview anymore. The changes will be reverted *soon:tm:*.

But, If anyone is interested in updating the preview, I'm willing to try and reintroduce grid adjustment to Lawnchair 15 (as I already have it implemented but the icon doesn't scale well on adjustment)

if interested to what happened to the phone, I separated the screen from the device! How amazing! Comes with tons of cracks! Lastly, I broke the display connector!

File: CONTRIBUTING.md (hunk 1 above)
Comment (validcube) [reply CONTRIBUTING.md]: 👋 Hello! (this message was almost 8 hours ago, I just never hit send)

@Goooler The change is intended to be consistent, but looks redundant. I believe @SuperDragonXD suggestion works well. What do you think?

**Comment by 0xAbdiiii** [comment]
@validcube does the preview have to be exactly like the one already present? it feels off because icon packs don't apply.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Goooler) [reply CONTRIBUTING.md]: Looks good.

**Comment by validcube** [comment]
> @s0m3th1ng-r4nd0m @validcube does the preview have to be exactly like the one already present? it feels off because icon packs don't apply.

The preview will most likely match the one from LC14 (see https://github.com/LawnchairLauncher/lawnchair), the configuration doesn't have to be the exact same, the frame is _aspirational_.

**Comment by validcube** [comment]
This PR is finished, preview has been updated to LC15.

**Review by Goooler — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -125,3 +126,9 @@ Strings `names` in `strings.xml` should follow this format:
...
 | Feature string                                   | (feature_name)_$1 | `colorpicker_hsb`          | HSB                  | Feature strings are strings that are confined to a specific feature. Examples include the gesture and color picker. |
 | Launcher string                                  | $1_launcher       | `device_contacts_launcher` | Contacts from device | Strings that are specific to the Launcher area                                                                      |
+
+#### Updating locally stored font listing
+
+Lawnchair uses a locally stored JSON file (`google_fonts.json`) to list available fonts from Google Fonts. This file should be updated periodically or before release to include the latest fonts.
+
+To update the font listing on Lawnchair, follow a step to [acquire your Google Fonts Developer API key](https://developers.google.com/fonts/docs/developer_api#APIKey), once you've got your API key, append `?key=API_KEY` to https://www.googleapis.com/webfonts/v1/webfonts then replace the content of [`google_fonts.json`](lawnchair/assets/google_fonts.json) with the response of the API.
```
Comment (SuperDragonXD) [inline CONTRIBUTING.md]:
```suggestion
To update Lawnchair's font listing, follow these steps:
1. Acquire a [Google Fonts Developer API key](https://developers.google.com/fonts/docs/developer_api#APIKey).
2. Download the JSON file from `https://www.googleapis.com/webfonts/v1/webfonts?key=API_KEY`, replacing `API_KEY` with the API key from step 1.
3. Replace the content of [`google_fonts.json`](lawnchair/assets/google_fonts.json) with the API response.
```

File: README.md [hunk 3]
```diff
@@ -17,23 +17,65 @@
...
+      <img alt="Get it on GitHub" src="docs/assets/badge-github.png" height="60">
+    </picture>
+  </a>
+</p>
+
+Lawnchair on the Play Store will install as a different app from other sources and features may be restricted to comply with the publishing rules.
+
+### Development builds
+
+Interested in keeping yourself up-to-date with every Lawnchair development? Try our development build, these builds offer the latest features and bug fixes at a cost of being slower and *might* introduce new bugs that have never been seen in stable. If you'd like to join in the fun, don't forget to make backup(s) and proceed with cautions.
```
Comment (SuperDragonXD) [inline README.md]:
```suggestion
Interested in keeping yourself up-to-date with every Lawnchair development? Try our development build!

These builds offer the latest features and bug fixes at a cost of being slower and introducing new bugs. Ensure that you make backups before installing!
```

File: README.md [hunk 4]
```diff
@@ -17,23 +17,65 @@
...
+  </a>
+    <a href="https://github.com/LawnchairLauncher/lawnchair/releases/latest">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="docs/assets/badge-github.png" height="60">
+      <img alt="Get it on GitHub" src="docs/assets/badge-github.png" height="60">
+    </picture>
+  </a>
+</p>
+
+Lawnchair on the Play Store will install as a different app from other sources and features may be restricted to comply with the publishing rules.
```
Comment (SuperDragonXD) [inline README.md]:
```suggestion
Lawnchair on Play Store will install as a different app from other sources. Sone features may be restricted to comply with Google Play's publishing rules.
```

File: README.md [hunk 5]
```diff
@@ -17,23 +17,65 @@
...
--   Pixel Launcher-like UI with Material You design
--   At a Glance widget with Smartspacer support
--   QuickSwitch support
--   Global Search support
--   Icons, fonts, and colors customization
+-   **Material You Theming:** Adapts to your wallpaper and system theme.
+-   **At a Glance Widget:** Get information *At a Glance* with support for [Smartspacer](https://github.com/KieronQuinn/Smartspacer) extensions.
+-   **QuickSwitch Support:** Seamlessly integrate itself as recent apps provider (requires root on Android 10 and newer).
+-   **Global Search:** Find apps, contacts, and web results quickly from your home screen.
+-   **Highly customizable:** Customize Icons, fonts, and colors to your liking
```
Comment (SuperDragonXD) [inline README.md]: This doesn't seem to follow a parallel structure. Here's a suggestion to make it more consistent:

```suggestion
-   **Material You Theming:** Adapts to your wallpaper and system theme.
-   **At a Glance Widget:** Displays information *at a glance* with support for [Smartspacer](https://github.com/KieronQuinn/Smartspacer) extensions.
-   **QuickSwitch Support:** Integrates with Android Recents on Android 10 and newer. (requires root)
-   **Global Search:** Allows quick access to apps, contacts, and web results from the home screen.
-   **Customization Options:** Provides options to tweak icons, fonts, and colors to your liking.
```

I've also edited it to sound less like marketing. Such content would fit at our website instead.

File: SECURITY.md [hunk 6]
```diff
@@ -1,30 +1,37 @@
 # Security Policy
 
+<picture>
+    <!-- Avoid image being clickable with slight workaround --->
+    <source media="(prefers-color-scheme: dark)" srcset="docs/assets/lawnchair-round.webp" width="100">
+    <img alt="" src="docs/assets/lawnchair-round.webp" width="100">
+</picture>
```
Comment (SuperDragonXD) [inline SECURITY.md]: I don't think we need to have an icon for a page detailing security policies and measures.

File: systemUI/README.md [hunk 7]
```diff
@@ -1,3 +1,5 @@
 # SystemUI Module
 
-https://android.googlesource.com/platform/frameworks/base/+/refs/heads/main/packages/SystemUI/
\ No newline at end of file
+This directory contains all of the required SystemUI modules, such as Common, which serves as a helper for other SystemUI modules like Unfold for handling devices with hinges.
```
Comment (SuperDragonXD) [inline systemUI/README.md]:
```suggestion
This directory contains all of the required SystemUI modules. Some examples of modules include `Common`, which serves as a helper for other SystemUI modules, and `Unfold`, which handles devices with hinges.
```

File: systemUI/README.md [hunk 8]
```diff
@@ -1,3 +1,5 @@
...
+
+See: https://android.googlesource.com/platform/frameworks/base/+/refs/heads/main/packages/SystemUI/
```
Comment (SuperDragonXD) [inline systemUI/README.md]:
```suggestion
Upstream source code: https://android.googlesource.com/platform/frameworks/base/+/refs/heads/main/packages/SystemUI/
```

File: README.md [hunk 9]
```diff
@@ -17,23 +17,65 @@
     <img alt="Google Pixel running Lawnchair Launcher with green wallpaper" src="docs/assets/device-frame.png" width="250px">
 </picture>
 
-Lawnchair is a free, open-source home app for Android. Taking Launcher3 — Android’s default home app — as a starting point, it ports Pixel Launcher features and introduces rich options for customization.
+Lawnchair is a free, open-source home app for Android. Taking Launcher3—Android’s default home app—as a starting point, it ports Pixel Launcher features and introduces rich customization options.
 
-This branch houses the codebase of Lawnchair 14, which is currently in beta and is based on Launcher3 from Android 14. For Lawnchair 9 to 13, see the branches with the `9-` to `13-` prefixes, respectively.
+This branch houses the codebase of Lawnchair 15, which is currently in alpha and is based on Launcher3 from Android 15. For Lawnchair 9 to 14, see the branches with the `9-` to `14-` prefixes, respectively.
```
Comment (SuperDragonXD) [inline README.md]: Once we're in beta, it doesn't really make sense to return back to alpha. I'd suggest changing this to `beta`, or more accurately, in `development` (as we haven't released a new version yet). We can change this to `beta` once we've released a new version.

File: build.gradle [hunk 10]
```diff
@@ -136,9 +136,9 @@ final def quickstepMaxSdk = "35"
 android {
     namespace "com.android.launcher3"
     defaultConfig {
-        // Lawnchair Launcher 14.0 Beta 3
+        // Lawnchair Launcher 15.0 Alpha 1
         // See CONTRIBUTING.md#versioning-scheme
-        versionCode 15_00_00_00
+        versionCode 15_00_01_01
```
Comment (SuperDragonXD) [inline build.gradle]: See my comment above. As this is a technical matter, we can use Lawnchair 15.0 Beta 01 for this.

**Review by SuperDragonXD — CHANGES_REQUESTED** [review]

**Comment by SuperDragonXD** [comment]
I'll note that text should follow Google's [Developer Documentation Style Guide](https://developers.google.com/style) as much as possible.

**Review by SuperDragonXD — APPROVED** [review]

**Review by MrSluffy — APPROVED** [review]

**Pull request merged**